HelpSource/scdoc.css

```diff
@@ -55,49 +60,14 @@ input {
     max-width: 100%;
 }
 
-#inheritedclassmets, #inheritedinstmets {
-    margin-top: 1em;
-}
-
-.inheritedmets_class {
-    font-weight: bold;
-    color: #555;
-    margin-top: 0.5em;
-    margin-bottom: 0.25em;
-}
-.inheritedmets_class a {
-    color: #333;
-}
-ul.inheritedmets {
-    list-style: none;
-    padding: 0;
-    margin: 0;
-    margin-left: 2em;
-    margin-top: 0.25em;
-}
-ul.inheritedmets li {
-    font-family: monospace;
-}
-a.inheritedmets_toggle {
-    font-size: 9pt;
-    color: #558;
-    font-weight: normal;
-/*    margin-left: 2em;*/
-}
-
-a.inheritedmets_toggle:hover {
-    text-decoration: none;
-    color: #000;
-}
-
```

Comment:
I understand you wanted to simplify the colors, but why did you remove all these rules? As a results inherited methods look less readable to me, as they get closer together since they lose some spacing.

Reply:
Good catch - I reintroduced these lines back again.